.github/workflows/e2e.yml

```diff
@@ -58,6 +66,14 @@ jobs:
           - 5432:5432
         # needed because the postgres container does not provide a healthcheck
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+      minio:
+        image: minio/minio:edge-cicd
```

Comment:
I initially wanted to use this image as well in https://github.com/opencollective/opencollective-api/pull/10688, but it [hasn't been updated in 3 years](https://hub.docker.com/layers/minio/minio/edge-cicd/images/sha256-29e8e51691d11e779468f275002779b221fd3902518d103e35c8a8bb2ef0f3ea) and that was causing some issues in my tests (I don't remember where), which is why I went with the `npm run minio &` approach.

No blocker for me if the image is working here, we can always update later if needed.